rumcake/src/hw/mod.rs

```diff
@@ -123,6 +127,22 @@ pub async fn output_switcher() {
     }
 }
 
+#[cfg(feature = "bootloader-double-tap-reset")]
+pub static BOOTLOADER_MAGIC: u32 = 0xDEADBEEF;
```

Comment:
I think `const` makes more sense here (also removing `pub` following the other suggestion)

```suggestion
const BOOTLOADER_MAGIC: u32 = 0xDEADBEEF;
```